src/Management/test/Endpoint.Test/Actuators/CloudFoundry/StartupWithSecurity.cs

```diff
@@ -22,4 +25,15 @@ public void ConfigureServices(IServiceCollection services)
     public void Configure(IApplicationBuilder app)
     {
     }
+
+    private sealed class MockHttpClientFactory(MockHttpMessageHandler mockHttpMessageHandler) : IHttpClientFactory
+    {
+        private readonly MockHttpMessageHandler _mockHttpMessageHandler =
+            mockHttpMessageHandler ?? throw new ArgumentNullException(nameof(mockHttpMessageHandler));
+
+        public HttpClient CreateClient(string name)
+        {
+            return _mockHttpMessageHandler.ToHttpClient();
```

Comment:
It would be cleaner to use `DelegateToMockHttpClientHandler`, see example usage [here](https://github.com/SteeltoeOSS/Steeltoe/blob/main/src/Discovery/test/Eureka.Test/EurekaDiscoveryClientTest.cs#L289) (with HttpClientFactory).

The advantage is that the test doesn't need to know the name of the `HttpClient`, so the constant doesn't need to be public.

Reply:
The constant is in an internal class, so it already isn't _actually_ public... Regardless, this change does not have any impact on the accessor b/c it's still needed in `EndpointServiceCollectionExtensions.cs`, but it'll now use `DelegateToMockHttpClientHandler`